tools/yaml-templates/ios-test.yml

```diff
@@ -87,6 +87,13 @@ steps:
       script: 'nohup pnpm start-test-app --server-options-cert $(localhostSSLCertificate.secureFilePath) --server-options-key $(localhostSSLKey.secureFilePath) &'
       workingDirectory: '$(ClientSdkProjectDirectory)'
 
+  - task: CodeQL3000Init@0
+    inputs:
+      ${{ if eq(variables['Build.SourceBranch'], 'refs/heads/main') }}:
```

Comment:
According to codeQL, it should only run default branch (master or main)

Reply:
I don't mind setting it explicitly but I think codeQL by default checks branch. That is to say, in your branch, codeQL would not be triggered (because it isn't running on main) maybe and this inputs setting can be saved?

Also I cannot think of other way we can do something except checking this in and then monitor main branch to see if it fails or not. Hopefully it is not.

Reply:
CodeQL will by default check for changes to `main`. Using the inputs tag here is to add additional branches. I would recommend modifying this to look for both `main` and `release/*` branches instead of just main.

Reply:
ok, I think you guys have a point and I also observed the skipping of run on non-default branch. Let me remove it since it doesn't seem doing anything better.